README.md

```diff
@@ -53,10 +53,7 @@ All packages must contain the following scripts to perform the stated actions:
 | ✅            | `start`             | Runs complete test and build suite and starts the dev server.                                 |
 | ✅            | `serve`             | Runs build suite and starts the dev server. When serving `core`, takes optional `open` value as argument to open specific page. (e.g: `npm run serve core --open=bot`)      |
 | ✅            | `build`             | Runs build suite and outputs the result into `dist`. Takes optional `base` value as argument. |
-| ✅            | `prettify`          | Runs `prettify` script packages(s) to format code as per the specs of the package.            |
```

Comment:
We still have `npm run prettify` so we could just update the text for this a bit?

Reply:
We do, but this is mentioning the scripts that should be included in each package, which we don't need anymore.